R/archivr.R

```diff
@@ -143,41 +124,39 @@ list_string <- function (url_list) {
   return (paste0("'[", string, "]'"))
 }
 
-#' Saves a single url in either perma.cc or the wayback machine.
+#' Saves a single url in perma.cc.
 #'
 #' @param arc_url The url to archive.
-#' @param method Either "perma_cc" or the default, "wayback."
 #' @importFrom jsonlite fromJSON
 #' @import curl
 #' @export
 #' @return A list or object representing the result.
-archiv_url <- function (arc_url, method="perma_cc") {
+archiv_perma <- function (arc_url, method="perma_cc") {
```

Comment:
Remove "method" from the function parameters and add a local function instead.

Reply:
Don't understand the "add a local function" part instead. `method` isn't used at all, this was simply left in there by mistake.

Reply:
Oh I knew that some functions use method to call another.  So I was suggesting you just use `method <- "perma_cc"` instead. But I see that's not necessary.